java/jdbc/SpringBootSample/src/main/java/com/oracle/springapp/dao/AllTablesDAO.java

```diff
@@ -0,0 +1,14 @@
+package com.oracle.springapp.dao;
+
+import java.util.List;
+
+import com.oracle.springapp.model.AllTables;
+
+/**
+ * Simple DAO interface for EMP table.
```

Comment:
Is this for AllTables or EMP table?

Reply:
This is for AllTables. I have a separate one for EmployeeDAO.java

Reply:
Exactly my point; the comment says "Simple DAO interface for EMP table"; this is a copy/paste typo.